quickwit/quickwit-metastore/src/metastore/grpc_metastore/grpc_adapter.rs

```diff
@@ -288,10 +289,27 @@ impl grpc::MetastoreApiService for GrpcMetastoreAdapter {
             .0
             .add_source(&add_source_request.index_id, source_config)
             .await
-            .map(|_| SourceResponse {})?;
+            .map(|_| SourceResponse { has_changed: true })?;
```

Comment:
That's a pretty invasive change just to be able to tell the user that the source was already enabled/disabled. No worth it IMO. Let's not do it.